.github/workflows/create_agenda.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     - uses: pkgjs/meet
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
-        schedules: 2020-05-14T17:00:00.0Z/P4W
+        schedules: 2022-01-18T16:00:00.0Z/P1M
```

Comment:
A period of 1 month should inevitably lead to weekend meetings. Probable better revert to an integer number of weeks.